operator/data/profiles/remote.yaml

```diff
@@ -1,33 +1,6 @@
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
-  components:
-    pilot:
-      enabled: true
-    policy:
-      enabled: false
-    telemetry:
-      enabled: false
-    proxy:
-      enabled: false
-    sidecarInjector:
-      enabled: false
-    citadel:
-      enabled: false
-    galley:
-      enabled: false
-    cni:
-      enabled: false
-
   addonComponents:
     prometheus:
-      enabled: false
-
-  values:
-    security:
-      createMeshPolicy: false
-
-    global:
-      istioRemote: true
-      enableTracing: false
-      network: ""
+      enabled: false
```

Comment:
I think we should remove this and remote should inherit from the default profile. If Prometheus is a separate setup step for the master cluster it should also be a separate step in the remote. @mandarjog @costinm, wdyt? With this change we could remove the remote profile.

Reply:
don't default profile has istioRempte=false?  also, what about the network configuration for network 1 and network 2?

Reply:
I don't think we need istioRemote. Checking remote{Pilot,Policy,Telemetry}Address should be sufficient.

Reply:
Ok that SG!  thx!